core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -406,10 +407,34 @@ public static Duration getDuration(PropertyKey key) {
   }
 
   /**
-   * @return a view of the internal {@link Properties} of as an immutable map
+   * @return a map of the resolved properties(include default properties)
    */
   public static Map<String, String> toMap() {
-    return Collections.unmodifiableMap(PROPERTIES);
+    Map<String, String> map = toRawMap();
+    // Resolve values with ${VALUE} format
+    String nestedPropIdentifier = "$";
+    for (Map.Entry<String, String> entry : map.entrySet()) {
+      String value = entry.getValue();
+      if (value.contains(nestedPropIdentifier)) {
+        map.put(entry.getKey(), lookup(value));
+      }
+    }
+    return map;
+  }
+
+  /**
+   * @return a map of the raw properties(include default properties)
```

Comment:
"a map of the raw properties represented by this configuration, including all default properties"

Reply:
Done, thanks